docs/docs/how-gatsby-works-with-github-pages.md

```diff
@@ -77,3 +77,52 @@ After running `npm run deploy` you should see your website at `username.github.i
 If you use a [custom domain](https://help.github.com/articles/using-a-custom-domain-with-github-pages/), don't add a `pathPrefix` as it will break navigation on your site. Path prefixing is only necessary when the site is _not_ at the root of the domain like with repository sites.
 
 **Note**: Don't forget to add your [CNAME](https://help.github.com/articles/troubleshooting-custom-domains/#github-repository-setup-errors) file to the `static` directory.
+
+### Deploying to GitHub pages from Travis CI
+
+When using `gh-pages` npm module to deploy from a Travis CI (or other build systems), we need to configure it with proper credentials so that `gh-pages` is able to push the new branch.
```

Comment:
The sentence was a bit hard to read IMO. Unsure if this is better 😛 

```suggestion
When we want to deploy our website through `gh-pages` through a CI server, in this example, we'll use Travis CI (any build system will suffice). We want to use [gh-pages npm module](https://www.npmjs.com/package/gh-pages) to do our deploy. We need to configure it with proper credentials so that `gh-pages` is able to push the new branch.

```